glue_jupyter/bqplot/scatter/layer_artist.py

```diff
@@ -264,7 +264,7 @@ def _update_visual_attributes(self, changed, force=False):
                     self.density_mark.vmax = self.state.cmap_vmax
 
                 if force or "stretch" in changed:
-                    self.density_mark.stretch = STRETCHES[self.state.stretch]()
+                    self.density_mark.stretch = self.state.stretch_object
```

Comment:
@astrofrog: should this be made dependent on version or availability of `stretch_object`, or glue-core version requirement be bumped to 1.17?

Reply:
I'll just bump the requirements to 1.17